[IMP] rental: update unavailability setting (#19507)
odoo/documentation: content/applications/sales/rental/configure_products/product_type.rst

Reviewer: This looks a little weird. Maybe indent as a list?

Reviewer 2: ```suggestion
To view all products available for rent in the database, navigate to :menuselection:`Rental app -->
```
Caught this, fixing if we can.

Odoo allows for two :ref:`product types <inventory/type/good-or-services>` when configuring a rental
product: Goods or Services.

- *Goods*: Includes physical items. Goods are removed from the company’s stock, delivered to the
  customer, and then returned.
- *Services*: Includes labor, non-physical, or a physical item that never leaves the company’s

>   stock, such as a hotel room, work station, or storage unit.

Reviewer 2: ```suggestion
  stock, such as a hotel room, workstation, or storage unit.
```

Other docs in this rental use "workstation," so just trying to keep consistent.

The following sections detail basic settings and app integration configurations for goods and
services.

.. seealso::
   - :doc:`products`
   - :doc:`labor_service_products`
   - :doc:`service_products`

.. _rental/product_type/configuration:

Configuration
=============

.. important::
   The *Configuration* menu changes if the **Inventory** or **Sales** apps are installed.

   - For the *Rental Transfers* setting, the **Inventory** app must be installed.
   - For the *PDF Quote Builder* and *Quotation templates* settings, the **Sales** app must be
     installed.

To configure default settings on rental products, navigate to :menuselection:`Rental app -->
Configuration --> Settings`.

.. image:: product_type/rental-settings-with-inventory.png
   :alt: A configured Setting page with the Inventory section displayed for the Rental app.

> In the *Pick up & Returns* section, the :guilabel:`Availabilties` setting sets what days and

Reviewer 2: ```suggestion
In the *Pick up & Returns* section, the :guilabel:`Availabilities` setting sets what days and
```
Misspelled word.

The following sections detail basic settings and app integration configurations for goods and
services.

.. seealso::
   - :doc:`products`
   - :doc:`labor_service_products`

>    - :doc:`service_products`

Reviewer: I like having these at the bottom of the page. Your mileage may vary.

.. _rental/product_type/configuration:

Configuration
=============

.. important::

>    The *Configuration* menu changes if the **Inventory** or **Sales** app are installed.

Reviewer: ```suggestion
   The *Configuration* menu changes if the **Inventory** or **Sales** apps are installed.
```
